generate.js

```diff
@@ -58,179 +69,350 @@ var main = module.exports = function(versions, tests, restore) {
         var sections = {};
         var testSections = {};
 
-        function createComment(paramsStruct, section, funcName, indent) {
```

Comment:
It might look like I've rewritten the entire file, but it's mostly whitespace changes (indentation).